src/CMakeLists.txt

```diff
@@ -9,13 +9,13 @@ if(WIN32)
     if(HAVE_ICONV)
         target_link_libraries(unshield iconv)
     endif()
-    target_link_libraries(unshield z)
+    target_link_libraries(unshield ${ZLIB_LIBRARIES})
```

Comment:
Is it needed to have ${ZLIB_LIBRARIES} for both libunshield and the unshield application?

Reply:
I haven't tested this, but I assumed it, as 'z' was stated for both executables before